docs/user-guide/cli-using-using-prompt-feature.md

```diff
@@ -1,5 +1,72 @@
 # Using the prompt feature
 
-Zowe CLI has a command-line "prompt" feature that asks you to provide required option values. The CLI always prompts for host, port, username, and password if you do not supply them in commands or profile configuration.
+Zowe CLI uses a command-line "prompt" feature to ask you to provide required option values. The CLI always prompts for host, port, username, and password information if not supplied in commands or profile configuration.
 
-You can also manually enable the prompt for any option. This is helpful to mask sensitive information on the screen while you type. You can enable a one-time prompt, or choose to always prompt for a particular option.
\ No newline at end of file
+You can also manually enable the prompt for any option. This is helpful to mask sensitive information on the screen while you type. You can enable a one-time prompt, or choose to always prompt for a particular option.
+
+### Enabling a one-time prompt
+
+To enable a one-time prompt:
+
+1. Open the Zowe CLI command prompt.
+
+2. Specify an option or positional as `PROMPT*`:
```

Comment:
@awharn What do we mean by "positional"?

Reply:
These are the positional arguments, like `<data set name>`, in the [`zowe files list ds`](https://docs.zowe.org/stable/web_help/index.html?p=zowe_zos-files_list_data-set) command

Reply:
updated doc to say "positional arguments"